esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -145,7 +145,7 @@ struct WsWUExceptions
 #define WUINFO_All                      0xFFFFFFFF
 
 static constexpr unsigned defaultMaxLogRecords = 10000;
-static constexpr unsigned defaultWULogSearchTimeBufferSecs = 600;
+static constexpr unsigned defaultWULogSearchTimeBufferSecs = 86400;
```

Comment:
as I understand it, this is a workaround to cater for unknown timestamps.
It is expanding the time range the zap log collection collects, by increasing the workunit time period by 24hrs either side.

I think we need a comment here explaining this is a workaround + a new JIRA to revisit this to avoid this.
JIRA now opened : https://track.hpccsystems.com/browse/HPCC-27321

The issue is that the TZ from the HPCC logs is unknown (not noted in the logs themselves).

@rpastrana - could we assume (if TZ is absent from logs, i.e. for now) - that their TZ (and the workunit timestamp TZ) is the same TZ as the component collecting the logs?
i.e. that assumes that esp is on same TZ as all other components, which 99% of the time it will be.

Reply:
In offline chat, it has been suggested +/- 12hrs may be more prudent (reducing the query time potentially).

And hopefully this is a relatively short-term workaround, until HPCC/pod TZ's vs ALA TZ can be normalized.